File: web-front-mvc/src/main/java/template/shared/actuator/ActuatorHttpExchangesConfig.java
package template.shared.actuator;

import org.springframework.boot.actuate.web.exchanges.HttpExchangeRepository;
import org.springframework.boot.actuate.web.exchanges.InMemoryHttpExchangeRepository;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

@Configuration
public class ActuatorHttpExchangesConfig {

    private static final int EXCHANGE_CAPACITY = 2000;

    @Bean
    public HttpExchangeRepository httpExchangeRepository() {
        var repository = new InMemoryHttpExchangeRepository();
        repository.setCapacity(EXCHANGE_CAPACITY);
        return repository;
    }

}

File: web-front-reactive2/src/main/kotlin/template/shared/actuator/ActuatorHttpExchangesConfig.kt
package template.shared.actuator

import org.springframework.boot.actuate.web.exchanges.HttpExchangeRepository
import org.springframework.boot.actuate.web.exchanges.InMemoryHttpExchangeRepository
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration

@Configuration
class ActuatorHttpExchangesConfig {
    private val EXCHANGE_CAPACITY: Int = 2000

    @Bean
    fun httpExchangeRepository(): HttpExchangeRepository? {
        val repository = InMemoryHttpExchangeRepository()
        repository.setCapacity(EXCHANGE_CAPACITY)
        return repository
    }

}